src/Symfony/Component/Filesystem/Path.php

```diff
@@ -0,0 +1,990 @@
+<?php
+
+/*
+ * This file is part of the webmozart/path-util package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Filesystem;
+
+use function array_pop;
+use function array_slice;
+use function basename;
+use function ctype_alpha;
+use function explode;
+use function function_exists;
+use function getenv;
+use function implode;
+use function in_array;
+use InvalidArgumentException;
+use function is_object;
+use function ltrim;
+use function mb_strtolower;
+use function pathinfo;
+use function rtrim;
+use RuntimeException;
+use function sprintf;
+use function str_replace;
+use function strlen;
+use function strpos;
+use function substr;
+
+/**
+ * Contains utility methods for handling path strings.
+ *
+ * The methods in this class are able to deal with both UNIX and Windows paths
+ * with both forward and backward slashes. All methods return normalized parts
+ * containing only forward slashes and no excess "." and ".." segments.
+ *
+ * @since  1.0
```

Comment:
should be adapted/or removed

Reply:
the `@since` tags should be removed as they are not used in Symfony.